orchestra/project_api/views.py

```diff
@@ -74,6 +73,11 @@ def project_details_url(request):
 
 @api_endpoint(['GET'])
 def workflow_types(request):
-    workflows = WorkflowVersion.objects.all()
-    workflow_choices = {w.slug: w.name for w in workflows}
-    return {'workflows': workflow_choices}
+    workflows = {
+        w.slug: {
+            'name': w.name,
+            'versions': {v.slug: v.name for v in w.versions.all()}
+        }
+        for w in Workflow.objects.all()
+    }
```

Comment:
maybe let's do `{v.slug: {'name': v.name, 'description': v.description}}` for now.  this response is two-part:
- make the inner value a dict, for consistency with the parent element, and
- description seems useful for display.  i feel less strongly about this second point than i do abotu the first point :)